internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-collectd-mysql.discovery.yaml.tmpl

```diff
@@ -0,0 +1,55 @@
+{{ receiver "smartagent/collectd/mysql" }}:
+  rule:
+    docker_observer: type == "container" and port == 3306
+    host_observer: type == "hostport" and command contains "mysqld" and port == 3306
+    k8s_observer: type == "port" and pod.name contains "mysql" and port == 3306
+  config:
+    default:
+      type: collectd/mysql
+      username: {{ defaultValue }}
+      password: {{ defaultValue }}
+      databases:
+        - name: {{ defaultValue }}
+      isolatedCollectd: true
+  status:
+    metrics:
+      successful:
+        - strict: mysql_octets.rx
+          first_only: true
+          log_record:
+            severity_text: info
+            body: collectd/mysql receiver is working!
```

Comment:
```suggestion
            body: MySQL receiver is working!
```

Reply:
I went ahead and adopted this one when fixing up the rules.